package.json

```diff
@@ -43,7 +43,7 @@
     "start-image_viewer": "node devtools/image_viewer/server.js",
     "start": "npm run start-test_dashboard",
     "baseline": "node tasks/baseline.js",
-    "preversion": "check-node-version --node 10 --npm 6 && npm-link-check && npm ls --prod",
+    "preversion": "check-node-version --node 12.13 --npm 6.12.0 && npm-link-check && npm ls --prod",
```

Comment:
@archmoj I chose slightly more strict version checks this time.

Reply:
Now requesting for npm `6.13.x` https://github.com/plotly/plotly.js/pull/4322/commits/7759c7602be9d0247176d817bddf6ac0f6b5e630